cmd/runtimetest/main.go

```diff
@@ -342,6 +344,69 @@ func validateRootFS(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateRootfsPropagation(spec *rspec.Spec) error {
+	if spec.Linux == nil {
+		return nil
+	}
+
+	targetDir, err := ioutil.TempDir("/", "target")
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(targetDir)
+
+	switch spec.Linux.RootfsPropagation {
```

Comment:
I think we probably want a `default` case that logs an “unrecognized linux.rootfsPropagation: …” warning.